CHANGES.rst

```diff
@@ -1,16 +1,18 @@
 Changelog
 =========
 
-3.1.3 (unreleased)
-------------------
+3.2 (unreleased)
+----------------
 
 Breaking changes:
 
-- *add item here*
+- Change the image caption template to use ``<figure>`` and ``<figcaption>``.
+  [thet]
```

Comment:
if its is a breaking change, increase the major version number.

Reply:
done.